cardinal/server/handler/tx.go

```diff
@@ -69,15 +45,15 @@ func PostTransaction(
 		}
 
 		// extract the transaction from the fiber context
-		tx, fiberErr := extractTx(ctx, verify)
+		tx, fiberErr := extractTx(ctx, validator)
```

Comment:
It's idiomatic in Go to just reuse the err variable. You don't have to provide a different name for each error.

In this case, you can just do
`tx, err := extractTx(ctx, validator)`